source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -78,6 +84,37 @@ closes if there is an early termination:
 
 For more information on how to ensure a cursor closes, see the :ref:`cursor cleanup section <cursor_cleanup>`.
 
+Explain
+~~~~~~~
+
+Use the ``explain()`` method to determine the execution plan of an
+operation with the given verbosity level: 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Cursor.java
+   :language: java
+   :dedent:
+   :start-after: begin explainExample
+   :end-before: end explainExample
+
+Your output should look something like the following:
+
+.. code-block:: none
+   :copyable: false
+
+   [Document{{$cursor=Document{{queryPlanner=Document{{plannerVersion=1,
+   ...
+   winningPlan=Document{{stage=PROJECTION_SIMPLE, transformBy=Document{{stars=1, _id=0}}, 
+   inputStage=Document{{stage=COLLSCAN, filter=Document{{categories=Document{{$eq=Bakery}}}}, 
+   direction=forward}}}}, rejectedPlans=[]}}, executionStats=Document{{executionSuccess=true, 
+   nReturned=3, executionTimeMillis=0, 
+   ...
+
+.. include:: /includes/fundamentals/explain-verbosity.rst
+
+For more information about this section, see the
```

Comment:
Nit: I think "this section" could be made more specific.

Reply:
Initially, I didn't have "about this section" because looking at it here, it can only mean for the explain method. But with the table rendered and it also having a link to another page, it was a bit awkward to see both links in two different sentences.